spec/models/course_spec.rb

```diff
@@ -421,4 +405,67 @@
       end
     end
   end
+
+  describe "#sites_not_associated_with_course" do
+    let(:first_site) { create(:site) }
+    let(:second_site) { create(:site) }
+    let(:provider) { create(:provider, sites: [first_site, second_site]) }
+    let(:first_site_status) { create(:site_status, :findable, site: first_site) }
+    subject { create(:course, provider: provider, site_statuses: [first_site_status]) }
+
+    its(:sites_not_associated_with_course) { should eq([second_site]) }
+  end
+
+  context "adding and removing sites on a course" do
```

Comment:
At the risk of sounding picky ... can we make this a `describe`? We should reserve `context` to define the setup for a test, and `describe` is what is being tested. If that makes any sense. :)

Reply:
fixed